ios/Classes/AblyFlutter.h

```diff
@@ -25,14 +25,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 -(ARTPaginatedResult *) getPaginatedResult:(NSNumber *const) handle;
 
-/**
- This method must be called from the main dispatch queue. It may only be called
- once.
- 
- @param completionHandler Will be called on the main dispatch queue when all
- platform objects have been closed down cleanly.
- */
```

Comment:
I removed this comment because it is wrong. `disposeWithCompletionHandler` could (and **was**) being called more than once. Whenever the app hot restarts (Flutter app restarts but iOS app doesn't), this method was being called.

Then I remove the outdated @param, since i removed it, because it is no longer necessary.